src/util/events.rs

```diff
@@ -161,6 +161,12 @@ pub enum Event {
 		node_id: PublicKey,
 		/// The action which should be taken.
 		action: Option<msgs::ErrorAction>
+	},
+
+	/// Broadcast spendable outputs to user wallet
+	SpendableOutputs {
```

Comment:
Nit this should go in the first section with user-handled events not PeerManager-handled events.